packages/flutter/test/foundation/error_reporting_test.dart

```diff
@@ -70,18 +66,18 @@ Future<Null> main() async {
       '^══╡ EXCEPTION CAUGHT BY ERROR HANDLING TEST ╞═══════════════════════════════════════════════════════\n'
       'The following assertion was thrown testing the error handling logic:\n'
       'Message goes here\\.\n'
-      '\'[^\']+flutter${dividerRegExp}test${dividerRegExp}foundation${dividerRegExp}error_reporting_test\\.dart\': Failed assertion: line [0-9]+ pos [0-9]+: \'false\'\n'
+      '\'[^\']+flutter/test/foundation/error_reporting_test\\.dart\': Failed assertion: line [0-9]+ pos [0-9]+: \'false\'\n'
```

Comment:
(following up from the other PR:)

Huh. Do we know how Windows developers feel about this? It seems surprising that we wouldn't use platform conventions in filenames in our stack traces.

cc @goderbauer

Reply:
Note that when packages are involved Dart VM used to report errors like this `package:a\b\c` on Windows which is super confusing because `package` paths are supposed to be uris which use `/` not `\`.